plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/SqlServerQueryRunner.java

```diff
@@ -61,6 +62,7 @@ public static QueryRunner createSqlServerQueryRunner(
 
             queryRunner.installPlugin(new SqlServerPlugin());
             queryRunner.createCatalog(CATALOG, "sqlserver", connectorProperties);
+            log.info("%s catalog properties: %s", CATALOG, connectorProperties);
```

Comment:
It is not needed. It is already logged.

Reply:
This PR disables the regular `Bootstrap` logger that would log this. This has been added since the DB name is random. It gets logged as:
```
 2022-01-27T11:23:16.813-0600	INFO	pool-4-thread-1	io.trino.plugin.sqlserver.SqlServerQueryRunner	sqlserver catalog properties: {connection-*** connection-url=jdbc:sqlserver://localhost:49165;database=database_e6cbbe8278d54bd791b508e1ef187d3d;lockTimeout=60000, connection-user=sa, allow-drop-table=true}
```
You can see that there are no other occurrences of the db name in the CI log, except for failed query error messages.